src/Umbraco.Web.UI/Umbraco/config/lang/da.xml

```diff
@@ -1636,5 +1636,7 @@ Mange hilsner fra Umbraco robotten
     <key alias="openBackofficeSearch">Åben backoffice søgning</key>
     <key alias="openCloseBackofficeHelp">Åben/Luk backoffice hjælp</key>
     <key alias="openCloseBackofficeProfileOptions">Åben/Luk dine profil indstillinger</key>
+    <key alias="currentLanguage">Aktivt sprog:</key>
+    <key alias="switchLanguage">Skift sprog til:</key>

```

Comment:
@BatJan I would probably just add the colons in the view/controller instead. It would make it easier to reuse these keys in other areas.

Reply:
Yes, good point - Done now 👍